IronFrame.Test/Utilities/FirewallManagerTests.cs

```diff
@@ -0,0 +1,224 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using NetFwTypeLib;
+using Xunit;
+
+namespace IronFrame.Utilities
+{
+    public class FirewallManagerTests
+    {
+        private const string Username = "Administrator";
```

Comment:
You can create a dynamic user.  We do that in other tests that require a specific user.